### PR TITLE
Scoping

### PR DESCRIPTION
**Pull request by JakeSidSmith**
* Update package scope, name and links
* Add info about fork to readme
* Swap CircleCI for Travis
* Use tslint-config-dabapps

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -181,3 +175,18 @@ const MyComponent = () => <div />;
 
 export default connect(mapStateToProps)(React.memo(MyComponent));
 ```
+
+## Updating this fork from upstream
+
+Ensure you have added a remote upstream in git e.g.
+
+```shell
+git remote add upstream git@github.com:JakeSidSmith/react-shallow-renderer.git
```
Comment (jordaneb) [inline README.md]: should this be `dabapps/react-shallow-renderer`?

File: README.md [hunk 2]
```diff
@@ -181,3 +175,18 @@ const MyComponent = () => <div />;
...
+```
+
+Create a new branch (from master) and run the following to pull changes from [upstream](https://github.com/jakesidsmith/react-shallow-renderer):
```
Comment (jordaneb) [inline README.md]: and this

File: README.md (hunk 1 above)
Comment (JakeSidSmith) [reply README.md]: Nah, this is the pointer to the original lib.

File: README.md (hunk 2 above)
Comment (JakeSidSmith) [reply README.md]: As above. 😊

**Pull request merged**